unlock-app/src/components/interface/MetadataTable.tsx

```diff
@@ -130,11 +131,12 @@ export const MetadataTable: React.FC<MetadataTableProps> = ({
   }
 
   const isKeyValid = (metadata: KeyMetadata) => {
-    if (!metadata?.expiration) return false
-    if (metadata?.expiration.toLowerCase() === 'never') return true
-    const now = new Date().getTime()
-    const expiration = new Date(metadata?.expiration).getTime()
-    return expiration > now
+    if (!metadata?.expiration) return // expiration not present
```

Comment:
I am confused about _why_ the expiration would be missing?

Reply:
Non necessary, now the function recive directly `expiration` as parameter